### ISSUE TITLE
Timeout not wired in client unary call

### ISSUE THREAD

**Issue by carlschwabni** [ni/grpc-labview#193]
It appears that the timeout input isn't wired into an optional timeout in the "Client Unary Call.vim":

![image](https://user-images.githubusercontent.com/91745004/201784364-dd635500-e778-418a-8411-981191afaf55.png)

Version 0.5.0.1

**Comment by AndrewHeim** [comment]
Ha!  This affects #272, #320, and possibly #77.

I didn't see there was even an input for the timeout when I investigated #320 .  I will submit this as a pull request.

When I tested it, it solved that issue.

**Comment by AndrewHeim** [comment]
Your situation makes sense - when I moved to higher performance situations, I got the hang more often.  Moving from the dev environment to the runtime engine is consistent with that.

For my patch, I simply wired the timeout to the input terminal.  So the Timeout control is wired to both the DLL call it was wired to, and now the Wait on Occurrence as well. 

You're right that ideally it really shouldn't get stuck at all.  When it does, this fix allows it to escape (eventually, depending on the timeout).  That still generates a spurious error.  In the LabVIEW world that's easy enough to filter out, but I can see how it might be more problematic depending on how it's being called from TestStand.

It's also worth referencing #210.  Apparently grpc references shouldn't be reused.  In my testing, this also *appeared* to get rid of the problem - at least after just a few minutes of testing.  It does slow things down a little bit... but it shouldn't slow down anything _near where the race is_.  So it's possible my race condition theory is wrong and something else is going on.  And it's possible getting a new grpc session each time doesn't get rid of this error.  But it might be a clue.

**Comment by kt-jplotzke** [comment]
Thanks.  After a bit more testing, I'm fairly confident that this race condition is _not_ my issue.  I've created a test gRPC method which has a 1000 msec delay before it returns and I still hang at "Wait for Occurrence", which rules out a performance/speed-related issue.  I'm going to open a new issue for my problem.
